test/prob/test_fixture_ccdf_log.hpp

```diff
@@ -481,9 +492,11 @@ class AgradCcdfLogTestFixture : public ::testing::Test {
       calculate_gradients_3rdorder(expected_gradients3, ccdf_log_funct, x3);
       calculate_gradients_3rdorder(gradients3, ccdf_log, y3);
 
-      test_gradients_equal(expected_gradients1, gradients1);
-      test_gradients_equal(expected_gradients2, gradients2);
-      test_gradients_equal(expected_gradients3, gradients3);
+      test_gradients_equal(expected_gradients1, gradients1, 1e-3);
```

Comment:
The reference implementation gradients are quite bad. I had to use a relative tolerance of 1e-3 to get them to pass (finite difference worked with 1e-4).

It's stuff like `gamma_p` and `gamma_q` I think (https://github.com/stan-dev/math/issues/2006).